File: 387-first-unique-character-in-a-string/first-unique-character-in-a-string.py
class Solution:
    def firstUniqChar(self, s: str) -> int:
        seen=[]
        for index in range(len(s)):
            if s[index] not in s[index+1:] and s[index] not in seen:
                return index
            seen.append(s[index])
        return -1

File: 387-first-unique-character-in-a-string/first-unique-character-in-a-string.c
int firstUniqChar(char* s) {
    
    int arr[26];
    for(int i=0;s[i]!='\0';i++){
        int index=s[i]-97;
        arr[index]++;
    }

    for(int i=0;s[i]!='\0';i++){
        int index=s[i]-97;
        if(arr[index]==1){
            return i;
        }
    }
    
    return -1;
}